x/campaign/keeper/msg_server_update_total_supply_test.go

```diff
@@ -3,6 +3,8 @@ package keeper_test
 import (
 	"testing"
 
+	testkeeper "github.com/tendermint/spn/testutil/keeper"
```

Comment:
If we add the import latter, the `goimport` does not solve the import sort well. It will put the project imports into a new block line. We should fix it manually